src/main/charts/bitbucket/templates/_helpers.tpl

```diff
@@ -524,6 +524,57 @@ volumeClaimTemplates:
 {{ end }}
 {{ end }}
 
+{{- define "bitbucket.openSearchEnvVars" -}}
+{{- if and .Values.bitbucket.clustering.enabled (or .Values.opensearch.enabled .Values.opensearch.baseUrl) }}
+- name: SEARCH_ENABLED
+  value: "false"
+- name: PLUGIN_SEARCH_CONFIG_BASEURL
+  value: {{ if .Values.opensearch.baseUrl }}{{ .Values.opensearch.baseUrl }}{{ else if .Values.opensearch.enabled }}http://opensearch-cluster-master:9200{{ else }}{{ .Values.opensearch.baseUrl }}{{ end }}
```

Comment:
I think it would make more sense if `.Values.opensearch.enabled` would be `.Values.opensearch.install.enabled` or something expressing the fact that we have installed it ourselves.